### PR TITLE
skywalking-cli: Add version 0.10.0

### PR DESCRIPTION
**Pull request by silverkorn**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/swctl.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "description": "SkyWalking CLI is a command interaction tool for the SkyWalking user or OPS team",
+    "homepage": "https://github.com/apache/skywalking-cli/",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.apache.org/skywalking/cli/0.9.0/skywalking-cli-0.9.0-bin.tgz",
+            "hash": "sha512:9a0445650c1fc55c8cfc53080742842abc5b36120dfc2f6adf430fc7211049637de362fdabf653929a52b0495327ad4ecd5c803c47abc19ada088ec73f26a19b"
+        }
+    },
+    "bin": "swctl.exe",
```
Comment (rashil2000) [inline bucket/swctl.json:12]:
```suggestion
    "bin": "bin\\swctl.exe",
```

File: bucket/swctl.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "pre_install": [
+        "Move-Item \"$dir\\skywalking-cli-*\\bin\\*-win*\" \"$dir\\swctl.exe\"",
+        "Remove-Item \"$dir\\skywalking-cli-*\" -Recurse"
```
Comment (rashil2000) [inline bucket/swctl.json:15]:
```suggestion
        "Rename-Item \"$dir\\bin\\swctl-*-win*\" \"swctl.exe\"",
        "Remove-Item \"$dir\\bin\\swctl-*\""
```

**Comment by rashil2000** [comment]
Rename app to skywalking-cli.json

File: bucket/swctl.json (hunk 2 above)
Comment (silverkorn) [reply bucket/swctl.json:15]: The downloaded binaries are called `skywalking-cli-*`, but to keep it consistent with their build output and their documentation, I will rename it to `swctl.exe`

**Comment by silverkorn** [comment]
Ok, thanks!
I will adjust this soon

**Comment by silverkorn** [comment]
Sorry for the delay, bit busy at the moment. 😬 
Thanks for the adjustments, seems good to me!

**Pull request merged**